lib/src/view/game/game_list_detail_tile.dart

```diff
@@ -0,0 +1,306 @@
+import 'package:dartchess/dartchess.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:lichess_mobile/src/model/common/chess.dart';
+import 'package:lichess_mobile/src/model/game/archived_game.dart';
+import 'package:lichess_mobile/src/model/game/player.dart';
+import 'package:lichess_mobile/src/styles/lichess_colors.dart';
+import 'package:lichess_mobile/src/styles/lichess_icons.dart';
+import 'package:lichess_mobile/src/styles/styles.dart';
+import 'package:lichess_mobile/src/utils/l10n_context.dart';
+import 'package:lichess_mobile/src/view/game/game_common_widgets.dart';
+import 'package:lichess_mobile/src/view/game/game_list_tile.dart';
+import 'package:lichess_mobile/src/view/game/status_l10n.dart';
+import 'package:lichess_mobile/src/widgets/adaptive_bottom_sheet.dart';
+import 'package:lichess_mobile/src/widgets/board_thumbnail.dart';
+import 'package:lichess_mobile/src/widgets/user_full_name.dart';
+import 'package:timeago/timeago.dart' as timeago;
+
+/// A list tile that shows more detailed game info than [GameListTile].
+class GameListDetailTile extends StatelessWidget {
+  const GameListDetailTile({
+    required this.item,
+  });
+
+  final LightArchivedGameWithPov item;
+
+  Side get mySide => item.pov;
+
+  @override
+  Widget build(BuildContext context) {
+    final (game: game, pov: youAre) = item;
+    final me = youAre == Side.white ? game.white : game.black;
+
+    final customColors = Theme.of(context).extension<CustomColors>();
+
+    final dateStyle = TextStyle(
+      color: textShade(
+        context,
+        Styles.subtitleOpacity,
+      ),
+      fontSize: 12,
+    );
+
+    return GestureDetector(
+      onLongPress: () {
+        showAdaptiveBottomSheet<void>(
+          context: context,
+          useRootNavigator: true,
+          isDismissible: true,
+          isScrollControlled: true,
+          showDragHandle: true,
+          builder: (context) => GameContextMenu(
+            game: game,
+            mySide: mySide,
+            showGameSummary: false,
+          ),
+        );
+      },
+      onTap: () => openGameScreen(item.game, item.pov, context),
+      child: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 16.0).add(
+          const EdgeInsets.only(bottom: 8.0),
+        ),
+        child: LayoutBuilder(
+          builder: (context, constraints) {
+            return IntrinsicHeight(
```

Comment:
Why is `IntrisicHeight` needed here?

It should be avoided, as flutter doc says it is relatively expensive, so we probably don't want that in an infinite list.

Reply:
hmm I copied this from https://github.com/lichess-org/mobile/blob/33904fcfb0daa944159bc7be1013cfa0a1db7bf9/lib/src/view/game/game_list_tile.dart#L138

I think it's needed because we set the `BoardThumbnail` size based on a fraction of the screen width?

Reply:
TBH I don't remember why it was needed in the context menu 😅

Reply:
Seems like it was used to make sure that the `Column` containing all the data has the same height as the board - But look like we can get rid of `IntrinsicHeight` and instead wrap the `Column` inside a `SizedBox(height: boardSize)` instead :tada: